### ISSUE TITLE
Roundcube login broken

### ISSUE THREAD

**Issue by hjjg** [mail-in-a-box/mailinabox#562]
Hi Josh, hi Community,

I need some help debugging this issue. I'm not able to log into roundcube anymore. The frontend 
says "connection to storage server failed", the mail.log:
Sep 24 21:20:08 mail dovecot: imap-login: Disconnected (no auth attempts in 0 secs): user=<>, rip=::1, lip=::1, TLS, session=<EC2DHsnMGwANNNAAAAAAAAAAAAAAAAAB>

And the Roundcube errorlog:
[24-Sep-2015 21:20:08 +0200]: <fsasd9q3t> IMAP Error: Login failed for user@mail.org from 82.192.111.22. Could not connect to ssl://localhost:993: Unknown reason in /usr/local/lib/roundcubemail/program/lib/Roundcube/rcube_imap.php on line 198 (POST /mail/?_task=login?_task=login&_action=login)

We upgraded PHP to 5.6 from this PPA:
http://ppa.launchpad.net/ondrej/php5-5.6

This was done because ownCloud randomly crashed the php5-fpm service. I don't know if this is the problem - maybe it was broken before.
I've tried imap_debug and debug with roundcube but this doesn't give more output.

**Comment by rainulf** [comment]
Do you have a valid SSL cert? PHP 5.6 checks for a valid one, so that may be the cause of it.

**Comment by h8h** [comment]
That work for me
```$config['imap_conn_options'] = array( 
   'ssl'         => array( 
   'verify_peer'  => true, 
   'verify_depth' => 3, 
   'cafile'       => '/home/user-data/ssl/ssl_certificate.pem', 
  ), 
);
```
File: /usr/local/lib/roundcubemail/config/config.inc.php

Don't know if it is a good idea to manipulate that file. It's only necessary if you're using php 5.6. Therefore I don't think that this snippet will be merged into master branch, because Ubuntu's offical php version is php 5.5.

**Comment by hjjg** [comment]
@h8h Thanks, I just tested your workaround and - if I did nothing wrong - it does not work.
I've changed the permissions of /home/user-data/ssl/ssl_certificate.pem, because this file was readable by root only. This did not help. I also checked if the array is applied to the configuration - it seems to be.

I use the following packages:
php5 5.6.14+dfsg-1+deb.sury.org~trusty+1
openssl 1.0.2d-2+deb.sury.org~trusty+1
roundcube 1.1.2
